pyorient/ogm/element.py

```
@@ -24,11 +24,16 @@ def save(self):
         return self._graph.save_element(self.__class__, self._props, self._id)
 
     def __setattr__(self, key, value):
-        element_entry = type(self).__dict__.get(key, None)
-        if isinstance(element_entry, Property):
-            self._props[key] = value
-        else:
-            super(GraphElement, self).__setattr__(key, value)
+        try:
```

Please use `hasattr` rather than catching an exception. I know that `hasattr` is implemented using the exception route, but this doesn't have to be the case in future versions of python, and there is no need to confuse future readers of the code as to why an exception is being swallowed.